pgx-pg-sys/build.rs

```diff
@@ -812,13 +811,10 @@ fn apply_pg_guard(items: &Vec<syn::Item>) -> eyre::Result<proc_macro2::TokenStre
                 let abi = &block.abi;
                 for item in &block.items {
                     match item {
-                        ForeignItem::Fn(func) => {
-                            // Ignore other functions -- this will often be
-                            // variadic functions that we can't safely wrap.
-                            if let Ok(tokens) = PgGuardRewriter::new().foreign_item_fn(func, abi) {
-                                out.extend(tokens);
-                            }
-                        }
+                        ForeignItem::Fn(func) => out.extend(quote! {
+                            #[pgx_macros::pg_guard]
```

Comment:
This will probably slow user builds down, as now more macro generation has to happen after the build script. That said I'll try and address this in a follow-up, it should actually be pretty easy.